src/osd/scrubber/pg_scrubber.cc

```diff
@@ -191,6 +191,9 @@ void PgScrubber::initiate_regular_scrub(epoch_t epoch_queued)
     reset_epoch(epoch_queued);
     m_fsm->process_event(StartScrub{});
     dout(10) << "scrubber event --<< StartScrub" << dendl;
+    stringstream oss;
```

Comment:
I would now prefer (as I did in all new code):
m_osds->clog->debug() << fmt::format(...

Reply:
Sure. Let me change it in fmt::format.